tests/run/isolated_limited_api_tests.srctree

```diff
@@ -73,7 +73,10 @@ def raises():
 
 def decode(bytes b, bytearray ba):
     return b.decode("utf-8") + ba.decode("utf-8")
-    
+
+def cast_float(object o):
+    return float(o)
```

Comment:
It seems worth testing that this code actually works and doesn't just compile by accident. You can add (Python) doctests that try different values.

Reply:
It's a srctree file so it's probably just easiest to add a few assets in run.py above rather than doctests

Reply:
I added a few values to run.py